test/built-ins/Temporal/Instant/prototype/toZonedDateTimeISO/to-zoned-date-time-iso.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.instant.prototype.tozoneddatetimeiso
+description: Test time zone parameters.
+includes: [temporalHelpers.js]
+features: [Temporal]
+---*/
+
+const inst = Temporal.Instant.from("1976-11-18T14:23:30.123456789Z");
```

Comment:
Here I would recommend using `new Instant()`. The epoch time doesn't actually matter, so you could use a round number like
```suggestion
const inst = new Temporal.Instant(1_000_000_000_000_000_000n);
```